api/docs/release.dox

```diff
@@ -149,6 +149,9 @@ Further non-compatibility-affecting changes include:
  - Added type_is_read() API that returns true if a trace type reads from memory.
  - Added instr_num_memory_read_access() and instr_num_memory_write_access() that return
    the number of memory read and write accesses of an instruction respectively.
+ - Added several routines to the #dynamorio::drmemtrace::memtrace_stream_t interface
+   for drmemtrace analysis tools: get_output_cpuid(), get_workload_id(),
```

Comment:
How is get_output_cpuid related to the shard_id in parallel_shard_init?

Reply:
Oh I see the test. They're the same, as expected.

Reply:
Right, this was added previously (but not to this release list) so you're not seeing its docs in this PR: the docs do say it's the output cpu ordinal for dynamic schedules.